config.yaml

```diff
@@ -34,12 +34,24 @@ manager:
     admin_username: admin
     # If admin_password is left blank, a password will be generated for you and provided
     # to you at the end of the install process.
+    # IMPORTANT - In case of clustering with an external database, take note of the admin_password
     admin_password: ''
 
   monitoring_install: &monitoring_install
     # This will cause the monitoring components not to be installed if set to true
     skip_installation: true
 
+cluster:
+  # The cluster's master manager's IP to join to
+  # Must be in the same architecture as this manager:
+  #   I.E - all-in-one with all-in-one or external db with external db
+  # IMPORTANT - Make sure to update the admin_password's value with the correct
+  #             admin_password of the master manager
+  join_cluster: ''
```

Comment:
This name is confusing. I would've thought it'd be a boolean (as in - join_cluster - yes/no).

Reply:
This was to keep the consistency of the regular AIO join operation (cfy cluster join ...) and this join operation, but I agree...
I'll change that to master_ip, I think it has a more clear intent, would you agree?